### PR TITLE
new gst_ios_init h/m files

### PR DESCRIPTION
**Pull request by Cowprod**
From gstreamer binaries

### CODE REVIEW

File: ios/GstPlay/gst_ios_init.h [hunk 1]
```diff
@@ -15,6 +15,7 @@ G_PASTE(g_io_module_, G_PASTE(name, _load_static)) ()
  * You can also enable individual plugins. See gst_ios_init.c to see their names
  */
 
+#define GST_IOS_PLUGINS_GES
```
Comment (sdroege) [inline ios/GstPlay/gst_ios_init.h]: This one can be commented out too. That's basically the new GST_IOS_PLUGINS_EDITING

**Comment by sdroege** [comment]
Looks good except for that one comment. Also please squash all the commits into a single one with a descriptive commit message (e.g. "ios: Sync gst_ios_init.[mh] with latest version")

**Comment by Cowprod** [comment]
I'm learning git in the same time :) I try to do this
I just commit the comment, is it a probleme to do this AFTER the
pull-request creation ?

**Comment by sdroege** [comment]
You can just add a new commit and then push to the same branch. The pull request will be updated then.

For squashing all commits together you have to push --force though. You can squash them all together with git rebase --interactive

**Comment by sdroege** [comment]
You could've also pushed the changes into the same pull request. You don't need to provide a new one for every iteration.

**Comment by Cowprod** [comment]
Squashing the commits drove me crazy...I prefered end then pull-request ant
work on the rtsp latency issue

2016-03-28 19:06 GMT+02:00 Sebastian Dröge <notifications@github.com>:

**Comment by sdroege** [comment]
Ok :) Next time you have a question for a pull request and don't find the answer, just send me mail and I'll try to help

**Comment by Cowprod** [comment]
thank you for your help :)

I succeed in setting latency...to 0.
I have a 200/300 ms latency, it is good but we have better result with this
app (about 150ms) in the same conditions

https://itunes.apple.com/fr/app/rb-ultra/id906668027?mt=8

Any chance to get better latency ? decoding settings ?, we don't need
audio...

Do you need more information about source ?

Do I open a new issue for that point ?



2016-03-28 19:39 GMT+02:00 Sebastian Dröge <notifications@github.com>:

**Comment by sdroege** [comment]
That's something not really related to GstPlayer but generally about GStreamer. Best to ask on the mailing list: https://lists.freedesktop.org/mailman/listinfo/gstreamer-devel

But setting the latency to 0 is not a good idea. Depending on network situation this should have a corresponding value that maps to the jitter of the network.

And getting latency lower, the other bits in here are usually audio sinks (you have no audio? then you can ignore this) and video decoder latency (which ones are you using?). And you have 150ms latency with the same input stream? Can you share that stream?

That's all things you can write about in your mail btw, not answer here :)

**Comment by Cowprod** [comment]
I created an account and posted to
gstreamer-devel@lists.freedesktop.org ...with
more technicals details ...wait and see :) thanks for all

2016-03-28 20:23 GMT+02:00 Sebastian Dröge <notifications@github.com>:

**Comment by sdroege** [comment]
Did you try gst_player_set_audio_track_enabled() to disable audio btw? Disable it before gst_player_play()

**Comment by Cowprod** [comment]
I Will Try as soon as possible ! 


Envoyé de mon iPhone

**Comment by Cowprod** [comment]
done in my project, no error on build but I can't really appreciate the
effect on latency right now .
I have no full HD  around me and the auvidea encoding board's ouput accepts
only 1080 screens !

I posted more details on the hardware configuration on the mailing list

http://gstreamer-devel.966125.n4.nabble.com/RTSP-Latency-td4676606.html

I saw you posted 1.8 binaries, any change to do on then gst_ios_init files
?

2016-03-28 20:50 GMT+02:00 Emmanuel Imbernon <cowprod@gmail.com>:

**Comment by sdroege** [comment]
No changes needed, thanks!

**Comment by Cowprod** [comment]
no "visible" change for latency with gst_player_set_audio_track_enabled we
are style about 200 of latency .
Some tests have been done this morning in the lab with the same encoding
board but without the optional wifi board (with a standard access point)

Sorry original email from the lab is in french
If colors doesn't appear tell me:

gree gstplay
orange RBUltra
Red my comments in english

premier test ce matin :



GstPlay : latence mesurée 300 250 300 230 270 (les variations sont je pense
liées à la lecture

RB ultra (<--appstrore app)latence réglée à 100 ms (<--software latency
 selection) : 160 150 160

La différence entre les deux logiciels  est sensible  à l’image.

J’ai monté la qualité de l’encodage à 6000 car à 2000 (<--encoding
parameter on the auvidea board, no change on latency but best quality) la
qualité d’images en dynamique est mauvaise.  (macro blocs visibles en
mouvements)

Pas de dégradation de la latence à 6000 et amélioration notable des macros
blocs. Par contre le dés-entrelacement reste visible







2016-03-29 11:52 GMT+02:00 Emmanuel Imbernon <cowprod@gmail.com>:

**Comment by sdroege** [comment]
Put it as a reply on the mailing list :) You can definitely get lower latencies though, if this is with baseline h264 you can go down to 1-2 frames of latency plus a few milliseconds for network jitter.

**Comment by Cowprod** [comment]
done, it is tested with main h264, baseline is a little better for latency,
but tests are done in the same condidions with the both softwares

2016-03-30 16:00 GMT+02:00 Sebastian Dröge <notifications@github.com>:

**Pull request closed without merging**